ArcGISRuntimeSDKQt_CppSamples/Features/ListRelatedFeatures/ListRelatedFeatures.cpp

```diff
@@ -80,58 +91,56 @@ void ListRelatedFeatures::connectSignals()
   {
     if (!loadError.isEmpty())
       return;
-    for (int i = 0; i < m_map->operationalLayers()->size(); i++)
+
+    for (int i = 0; i < m_map->operationalLayers()->size(); ++i)
     {
       // get the Alaska National Parks layer
-      if (m_map->operationalLayers()->at(i)->name().contains(QString("- Alaska National Parks")))
+      if (m_map->operationalLayers()->at(i)->name().contains(QStringLiteral("- Alaska National Parks")))
       {
         m_alaskaNationalParks = static_cast<FeatureLayer*>(m_map->operationalLayers()->at(i));
+        m_alaskaFeatureTable = static_cast<ArcGISFeatureTable*>(m_alaskaNationalParks->featureTable());
+
+        // connect to queryRelatedFeaturesCompleted signal
+        connect(m_alaskaFeatureTable, &ArcGISFeatureTable::queryRelatedFeaturesCompleted,
+                this, [this](QUuid, QList<RelatedFeatureQueryResult*> rawRelatedResults)
+                {
+                  FeatureQueryListResultLock lock(rawRelatedResults);
+                  for (const RelatedFeatureQueryResult* relatedResult : lock.results)
```

Comment:
I guess we don't need `qAsConst` here because the member is by const ref?

Reply:
Agreed